apps/system/js/audio_channel_manager.js

```diff
@@ -0,0 +1,267 @@
+          'normal': PLAY_CURRENT_AND_NEW_CHANNEL,
+          'foregroundContent': PLAY_CURRENT_AND_NEW_CHANNEL,
+          'backgroundContent': -1,
+          'alarm': PAUSE_CURRENT_CHANNEL,
+          'system': PLAY_CURRENT_AND_NEW_CHANNEL,
+          'ringer': VOLUME_DOWN_CURRENT_CHANNEL,
+          'telphony': VOLUME_DOWN_CURRENT_CHANNEL,
+          'notification': PLAY_CURRENT_AND_NEW_CHANNEL,
+          'publicNotification': PLAY_CURRENT_AND_NEW_CHANNEL
+        },
+        'system': {
+          'normal': PLAY_CURRENT_AND_NEW_CHANNEL,
+          'foregroundContent': PLAY_CURRENT_AND_NEW_CHANNEL,
+          'backgroundContent': -1,
+          'alarm': PLAY_CURRENT_AND_NEW_CHANNEL,
+          'system': PLAY_CURRENT_AND_NEW_CHANNEL,
+          'ringer': PLAY_CURRENT_AND_NEW_CHANNEL,
+          'telphony': PLAY_CURRENT_AND_NEW_CHANNEL,
+          'notification': PLAY_CURRENT_AND_NEW_CHANNEL,
+          'publicNotification': PLAY_CURRENT_AND_NEW_CHANNEL
+        },
+        'ringer': {
+          'normal': PLAY_CURRENT_AND_NEW_CHANNEL,
+          'foregroundContent': PLAY_CURRENT_AND_NEW_CHANNEL,
+          'backgroundContent': -1,
+          'alarm': PLAY_CURRENT_AND_NEW_CHANNEL,
+          'system': PLAY_CURRENT_AND_NEW_CHANNEL,
+          'ringer': PAUSE_THEN_VIBRATE_CURRENT_CHANNEL,
+          'telphony': PLAY_CURRENT_AND_NEW_CHANNEL,
+          'notification': PLAY_CURRENT_AND_NEW_CHANNEL,
+          'publicNotification': PLAY_CURRENT_AND_NEW_CHANNEL
+        },
+        'telphony': {
+          'normal': PLAY_CURRENT_AND_NEW_CHANNEL,
+          'foregroundContent': PLAY_CURRENT_AND_NEW_CHANNEL,
+          'backgroundContent': -1,
+          'alarm': PAUSE_THEN_VIBRATE_CURRENT_CHANNEL,
+          'system': PLAY_CURRENT_AND_NEW_CHANNEL,
+          'ringer': VIBRATE_NEW_CHANNEL,
+          'telphony': -1,
+          'notification': VIBRATE_NEW_CHANNEL,
+          'publicNotification': PLAY_CURRENT_AND_NEW_CHANNEL
+        },
+        'notification': {
+          'normal': VOLUME_DOWN_NEW_CHANNEL,
+          'foregroundContent': VOLUME_DOWN_NEW_CHANNEL,
+          'backgroundContent': -1,
+          'alarm': PLAY_CURRENT_AND_NEW_CHANNEL,
+          'system': PLAY_CURRENT_AND_NEW_CHANNEL,
+          'ringer': PLAY_CURRENT_AND_NEW_CHANNEL,
+          'telphony': PLAY_CURRENT_AND_NEW_CHANNEL,
+          'notification': PLAY_CURRENT_AND_NEW_CHANNEL,
+          'publicNotification': PLAY_CURRENT_AND_NEW_CHANNEL
+        },
+        'publicNotification': {
+          'normal': VOLUME_DOWN_NEW_CHANNEL,
+          'foregroundContent': VOLUME_DOWN_NEW_CHANNEL,
+          'backgroundContent': -1,
+          'alarm': PLAY_CURRENT_AND_NEW_CHANNEL,
+          'system': PLAY_CURRENT_AND_NEW_CHANNEL,
+          'ringer': PLAY_CURRENT_AND_NEW_CHANNEL,
+          'telphony': PLAY_CURRENT_AND_NEW_CHANNEL,
+          'notification': PLAY_CURRENT_AND_NEW_CHANNEL,
+          'publicNotification': PLAY_CURRENT_AND_NEW_CHANNEL
+        }
+      };
```

Comment:
We will probably add a new type for the webrtc calls, it should be "voip" and maybe we can add it first. I will update the spec with Jenny once we confirm voip type is added.